README.rst

```diff
@@ -44,54 +44,6 @@ Furthermore, ooniprobe takes no precautions to protect the install target machin
 from forensics analysis.  If the fact that you have installed or used ooni
 probe is a liability for you, please be aware of this risk.
 
-OONI in 5 minutes
-=================
-
-On Debian testing or unstable::
-
-    sudo apt-get install ooniprobe
-
-On Debian stable (jessie)::
-
-    echo 'deb http://ftp.debian.org/debian jessie-backports main' | sudo tee -a /etc/apt/sources.list
-    sudo apt-get update
-    sudo apt-get install ooniprobe
-
-On Ubuntu 16.04 (xenial), 15.10 (wily) or 14.04 (trusty)::
-
-    echo 'deb http://deb.torproject.org/torproject.org $RELEASE main' | sudo tee -a /etc/apt/sources.list
-    sudo apt-get update
-    sudo apt-get install ooniprobe
-
-Note: You'll need to swap out ``$RELEASE`` for either ``xenial``, ``wily``, or
-``trusty``. This will not happen automatically. You will also need to ensure
-that you have the ``universe`` repository enabled. The ``universe`` repository
-is enabled by default in a standard Ubuntu installation but may not be on some
-minimal on not standard installations.
-
-On unix systems::
-
-    sudo pip install ooniprobe
-
-To install it from the current master run::
-
-    sudo pip install https://github.com/TheTorProject/ooni-probe/archive/master.zip
-
-Then run::
-
-    mkdir my_decks
-    oonideckgen -o my_decks/
-
-If you're using the Debian package, you will be asked when installing whether
-you would like to run OONI daily. On other platforms, if you would like to
-contribute measurements to OONI daily you can also add this to your crontab::
-
-    @daily ooniprobe $THE_OONI_COMMAND
-
-Run this command to automatically update your crontab::
-
-    (crontab -l 2>/dev/null; echo "@daily ooniprobe $THE_OONI_COMMAND") | crontab -
-
```

Comment:
Do we still need the crontabs?
The pip way to install ooniprobe is still in place, install subsection: `Unix systems (with pip)`.